docs/CHANGELOG.rst

```diff
@@ -12,7 +12,8 @@ Unreleased
 
 Added
 -----
-* Add Mus musculus to all BCM workflows' schemas 
+* Add Mus musculus to all BCM workflows' schemas
```

Comment:
Hmm... why does GitHub show this line as changed?

Reply:
Because I have my editor configured to remove all extra spaces at the end and the person who previously updated the changelog doesn't ;-) There is an extra space that got removed.